### ISSUE TITLE
Multiple image training

### ISSUE THREAD

**Issue by tischi** [embl-cba/fiji-plugin-cats#8]
@davodogster 
> How do I efficiently train a classifier on multiple images. E.g a 4 channel stack for 8 different images.

**Comment by tischi** [comment]
If they all have the same size in x and y the easiest is to combine them into an Hyperstack, where you "abuse" the time-dimension for your 8 images. Do you know how to do this? 
If you send me a few (cropped) images as a download link I can record a video for you how to do this.

**Comment by davodogster** [comment]
I think I know what you mean but don't know how to implement yet - I'll give it a try and let you know.

You mean I set the number of time periods and then I can annotate different ROIs at different times?

**Comment by tischi** [comment]
> You mean I set the number of time periods and then I can annotate different ROIs for different times?

Yes. If your images all have the same size and you do not have movies to start with that is the easiest option to train on multiple images.

If your images have different sizes you can still do it! I wrote some documentation:
https://github.com/embl-cba/fiji-plugin-cats#training-a-classifier-on-multiple-data-sets

**Comment by davodogster** [comment]
Thanks so much

Sam
